recipe/meta.yaml

```diff
@@ -38,26 +40,28 @@ requirements:
     - libkml
     - libpng
     - blosc
-    - libgdal 2.*   # with libgdal-3 one ImportTest fails
-    - arrow-cpp >=0.12.1
+    - libgdal >=2.3.*
```

Comment:
Has it been verified that libgdal-3 works?

Reply:
not sure .. I am changing the pinning to discard a libgdal conflict

Reply:
libgdal-3 should work now (I run it on my machine), though we're still on 2.4.x for our main published builds due to some packaging issues introduced in 3.